src/Agent.Plugins/GitCliManager.cs

```diff
@@ -175,23 +175,34 @@ public async Task<int> GitFetch(AgentTaskPluginExecutionContext context, string
             string progress = reducedOutput ? string.Empty : "--progress";
 
             // default options for git fetch.
-            string options = StringUtil.Format($"{forceTag} --tags --prune {progress} --no-recurse-submodules {remoteName} {string.Join(" ", refSpec)}");
+            string fetchString = $"--tags --prune --progress --no-recurse-submodules {remoteName} {string.Join(" ", refSpec)}";
 
+            // insert prune-tags if knob is false to sync tags with the remote
+            if(!AgentKnobs.DisableFetchPruneTags.GetValue(context).AsBoolean()){
```

Comment:
Please fix formatting

Reply:
What do you mean? The space inbetween 178 and 180?

Reply:
```suggestion
            if (!AgentKnobs.DisableFetchPruneTags.GetValue(context).AsBoolean())
            {
```

Reply:
Also, I think it makes sense to use ternary expression like we already do with `progress` parameter. Inserting doesn't read as easily